runtime-testsuite/test/org/antlr/v4/test/runtime/go/BaseTest.java

```diff
@@ -391,7 +391,7 @@ public String execModule(String fileName) {
 			ProcessBuilder builder = new ProcessBuilder(goPath, "run", modulePath,
 					inputPath);
 			builder.environment().put("GOPATH",
-					runtimePath + File.pathSeparator + tmpdir);
+					builder.environment().get("GOPATH") + ":" + runtimePath + File.pathSeparator + tmpdir);
```

Comment:
I think an empty path element defaults to the current directory. Working on a fix to avoid that.